specs/ContextMenuRequested.md

```diff
@@ -0,0 +1,711 @@
+    [uuid(f562a2f5-c415-45cf-b909-d4b7c1e276d3), object, pointer_default(unique)]
+    interface ICoreWebView2ContextMenuItemCollection : IUnknown
+    {
+        /// The number of context menu items contained in the ICoreWebView2ContextMenuItemCollection
+        [propget] HRESULT Count([out, retval] UINT32* value);
+
+        /// Gets the context menu item at the given index.
+        HRESULT GetValueAtIndex([in] UINT32 index,
+            [out, retval] ICoreWebView2ContextMenuItem** value);
+
+        /// Removes ContextMenuItem at the specified index
+        HRESULT RemoveValueAtIndex([in] UINT32 index);
+
+        /// Will insert the new ContextMenuItem at the index specified
+        HRESULT AddValueAtIndex([in] UINT32 index, [in] ICoreWebView2ContextMenuItem* value);
+    }
+
+    [uuid(76eceacb-0462-4d94-ac83-423a6793775e), object, pointer_default(unique)]
+    interface ICoreWebView2_4 : ICoreWebView2_3
+    {
+        /// Add an event handler for the ContextMenuRequested event.
+        /// ContextMenuRequested event is raised when a context menu is requested by the user
+        /// and the browser hasn't disabled context menu usage.
+        /// The host can use their own UI to create their own context menu using
+        /// the data provided in the API or can add to / remove from the default
+        /// context menu. If the host doesn't handle the event, Webview will
+        /// display the original context menu.
+
+        HRESULT add_ContextMenuRequested(
+            [in] ICoreWebView2ContextMenuRequestedEventHandler* eventHandler,
+            [out] EventRegistrationToken* token);
+
+        /// Remove an event handler previously added with add_ContextMenuRequested.
+        HRESULT remove_ContextMenuRequested(
+            [in] EventRegistrationToken token);
+
+        /// Add an event handler for the CustomItemSelected event.
+        /// CustomItemSelected event is raied when the user selects a context menu item that
+        /// was inserted by the developer and is not part of the default browser context menu.
+        /// The developer will include the logic for the appropriate data in this event handler.
+
+        HRESULT add_CustomItemSelected(
+            [in] ICoreWebView2CustomItemSelectedEventHandler* eventHandler,
+            [out] EventRegistrationToken* token);
+
+        /// Remove an event handler previously added with add_CustomItemSelected.
+        HRESULT remove_CustomItemSelected(
+            [in] EventRegistrationToken token);
+        
+    }
+
+    /// A continuation of the ICoreWebView2Environment interface.
+    [uuid(04d4fe1d-ab87-42fb-a898-da241d35b63c), object, pointer_default(unique)]
+    interface ICoreWebView2Environment : IUnknown
+    {
+        /// Create a ContextMenuItem object used for developers to insert new items into the default browser context menu
+        /// The Enabled and visible properties will default to true and the IsChecked property will only be used if the 
```

Comment:
Saying they *default* to true confused me because it sounds like they default to one value but can be changed to another. I think rather you're saying that enabled and visible are just always true for custom context menu items right?
```suggestion
        /// The IsEnabled and IsVisible properties are always true for custom context menu items. The IsChecked property will only be used if the 
```